src/routes/auth.route.js

```diff
@@ -0,0 +1,39 @@
+import { Router } from 'express';
+import { authController } from './../controllers/index.js';
+import { authMiddleware } from './../middlewares/index.js';
+import { catchError } from './../utils/catchError.js';
+
+export const authRouter = new Router();
+
+authRouter.post('/registration', catchError(authController.register));
+
+authRouter.get(
+  '/activation/:activationToken',
+  catchError(authController.activate),
+);
+
+authRouter.post('/login', catchError(authController.login));
+
+authRouter.post('/reset-password', catchError(authController.resetPassword));
+
+authRouter.post(
+  '/reset-password/:resetPasswordToken',
+  catchError(authController.changePassword),
+);
+
+authRouter.get('/profile', authMiddleware, catchError(authController.getUser));
```

Comment:
The `authMiddleware` is used here to protect the `/profile` route. Ensure that it correctly verifies the user's authentication status.